src/main/java/org/fundacionjala/coding/ovidio/movies/Customer.java

```diff
@@ -0,0 +1,69 @@
+package org.fundacionjala.coding.ovidio.movies;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+/**
+ *
+ */
+
+
+public class Customer {
+    private String name;
+    private Vector rentals = new Vector();
+
+    /**
+     * @param name receibe one number.
+     */
+    public Customer(String name) {
+        this.name = name;
+    }
+
+    /**
+     * @param arg recibe a rent.
+     */
+    public void addRental(Rental arg) {
+        rentals.addElement(arg);
+    }
+
+    /**
+     * @return name.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * @return statement.
+     */
+    public String statement() {
```

Comment:
This method is doing 3 things. it is calculating the totalAmount, totalFrequentRenterPoints and it is printing.
Please split it.